version.yaml

```
@@ -0,0 +1 @@
+logstash-output-dynatrace: '0.5.0'
```

We could consider
```suggestion
logstash-output-dynatrace: '0.5.0.rc1'
```
as per https://guides.rubygems.org/patterns/#prerelease-gems.